messaging/Kafka-producer/src/test/java/io/quarkus/ts/messaging/kafka/BlockingProducerIT.java

```diff
@@ -0,0 +1,98 @@
+package io.quarkus.ts.messaging.kafka;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.Duration;
+import java.util.Map;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.smallrye.mutiny.Uni;
+import io.smallrye.mutiny.helpers.test.UniAssertSubscriber;
+import io.vertx.ext.web.client.WebClientOptions;
+import io.vertx.mutiny.core.Vertx;
+import io.vertx.mutiny.ext.web.client.WebClient;
+
+@Tag("QUARKUS-1090")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Why is needed this annotation?

Reply:
// TODO https://github.com/quarkus-qe/quarkus-test-framework/issues/248
// Remove TestInstance annotation and after all once this ticket is resolved